### DISCUSSION TITLE
Changing Login and Sign-in Services

### DISCUSSION THREAD

**Discussion by gabrielkarras** [gabrielkarras/SOEN341#71] [General]
To summarize, @minhtrannhat wanted to rebuild a custom Login/Sign-in service.
This would allow the team easier access to the service's elements(i.e.: styling with CSS and interact with Django REST framework).
This was suggested because our current service works through a Django based API and would make it harder to work with React.

**Comment by minhtrannhat** [comment]
Thanks @gabrielkarras ! I'll leave this link here as reference: https://blog.devgenius.io/django-react-authentication-part-1-d351726b284d

**Comment by gabrielkarras** [comment]
Related to user story #38

**Comment by gabrielkarras** [comment]
We had issues with the django-auth module. So, we decided to ask the TA for feedback and login requirements.
